### PR TITLE
formatters/img: use new API for bounding box for Pillow 9.2

### PR DESCRIPTION
**Pull request by birkenfeld**

### CODE REVIEW

**Comment by birkenfeld** [comment]
TBH I just looked at https://pillow.readthedocs.io/en/stable/_modules/PIL/ImageFont.html#ImageFont.getsize which shows that all of the three methods (getsize, getbbox, getlength) use the same underlying API...

File: pygments/formatters/img.py [hunk 1]
```diff
@@ -206,13 +206,17 @@ def get_char_size(self):
...
 
     def get_text_size(self, text):
         """
-        Get the text size(width, height).
+        Get the text size (width, height).
         """
-        return self.fonts['NORMAL'].getsize(text)
+        font = self.fonts['NORMAL']
+        if hasattr(font, 'getbbox'):  # Pillow >= 9.2.0
+            return font.getbbox(text)[2:4]
```
Comment (Anteru) [inline pygments/formatters/img.py:217]:
```suggestion
            return font.getlength(text)
```

**Review by Anteru — APPROVED** [review]
What was the reason to pick `getbox` over `getlength`? The underlying API is the same and this avoids unpacking the tuple.

File: pygments/formatters/img.py (hunk 1 above)
Comment (birkenfeld) [reply pygments/formatters/img.py:217]: If I read the docs correctly, `getlength` only returns a single value, while `getsize` returns `(width, height)`.

**Pull request merged**